tools/dependency/cve_scan.py

```diff
@@ -7,11 +7,14 @@
 import datetime as dt
 import gzip
 import json
+import os
 import re
 import sys
 import textwrap
 import urllib.request
 
+os.environ["API_PATH"] = "external/envoy_api_canonical"
```

Comment:
Why load from here rather than the source tree directly?

Reply:
because its inside bazel - i figured that is how bazel works in terms of reproducible builds - i guess the flip question would be - why have an envoy_api_canonical target

Reply:
Oh right, even for source files we are going through a repository rule. This will be simplified when we remove v2 support.

In any case, I suggest maybe just in-lining this path in the imported Python module, rather than using environment variable tricks which are a bit more brittle..

Reply:
> In any case, I suggest maybe just in-lining this path in the imported Python module

using `args` and `$(location ...)`

i havent quite figured out an optimal pattern, but im avoiding args because it makes it harder for targets to use each otther (in my v limited experience)

Reply:
im going to merge this PR with the one to shift the external dep rst...